apps/aeutils/priv/aeternity_config_schema.json

```diff
@@ -1748,6 +1748,11 @@
                                             "default": 0,
                                             "type": "integer"
                                         },
+                                        "hyperchain_static_coinbase": {
+                                            "type": "integer",
+                                            "default": 0,
+                                            "description": "HC coinbase value for HyperChains consensus"
```

Comment:
Could you explain more, such as that this is the block production reward?

Reply:
And that this many new tokens will be minted for each block resulting in some inflation.

Reply:
We should probably not go too deep into it here and explain it better in the external documentation, but I think a nice message to the person configuring this would be good.

Reply:
Since I really have no exact idea what it's for, I hereby solicit suggestions of what this should read.

Reply:
@happi ???